code/modules/mob/living/carbon/human/species/station/prometheans_vr.dm

```diff
@@ -18,8 +18,10 @@
 		/mob/living/proc/set_size,
 		/mob/living/carbon/human/proc/succubus_drain,
 		/mob/living/carbon/human/proc/succubus_drain_finialize,
-		/mob/living/carbon/human/proc/succubus_drain_lethal
-		)
+		/mob/living/carbon/human/proc/succubus_drain_lethal,
+		/mob/living/carbon/human/proc/bloodsuck,
+		/mob/living/carbon/human/proc/shred_limb
```

Comment:
Yeeeeaaaahhh Prometheans aren't gonna get this one.

Reply:
I can remove the verbs from them. They were there mostly for "In case they want to use these fluff verbs" but I can do that in another PR later down the road, as to not conflict with what this current PR is really for.